azure-iot-hub-devicesdk/azure/iot/hub/devicesdk/transport/mqtt/mqtt_provider.py

```diff
@@ -17,7 +17,7 @@ class MQTTProvider(object):
     to publish/subscribe messages.
     """
 
-    def __init__(self, client_id, hostname, username, password, ca_cert=None):
+    def __init__(self, client_id, hostname, username, password, x509_info = None, ca_cert=None):
```

Comment:
>x509_info [](start = 64, length = 9)

Info is a horrible name.  It smells like you don't know what this object is.  for consistency, I think x509_info should maybe be x509_credentials.